From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@debian.org>
Date: Sun, 21 Jun 2020 18:42:51 +0200
Subject: Fix typo: evalute => evaluate

---
 daemons/controld/controld_control.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/daemons/controld/controld_control.c
+++ b/daemons/controld/controld_control.c
@@ -542,7 +542,7 @@
         "Zero disables polling, while positive values are an interval in seconds"
             "(unless other units are specified, for example \"5min\")",
         "15min", pcmk__valid_interval_spec,
-        "Polling interval to recheck cluster state and evalute rules "
+        "Polling interval to recheck cluster state and evaluate rules "
             "with date specifications",
         "Pacemaker is primarily event-driven, and looks ahead to know when to "
             "recheck cluster state for failure timeouts and most time-based "
